File: 백준/Bronze/11720. 숫자의 합/숫자의 합.c
#include <stdio.h>

int main()
{
    int N, answer = 0;
    scanf("%d", &N);
    char nums[N];
    scanf("%s", nums);
    
    for (int i = 0; i < N; i++) answer += nums[i] - '0';
    
    printf("%d", answer);
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int N = sc.nextInt();
		String nums = sc.next();
		int answer = 0;
		
		for (int i = 0; i < N; i++) answer += nums.charAt(i) - '0';
		
		System.out.println(answer);
	}
}